examples/minirocket.ipynb

```diff
@@ -120,12 +132,13 @@
    "source": [
     "We suggest using `RidgeClassifierCV` (scikit-learn) for smaller datasets (fewer than ~10,000 training examples), and using logistic regression trained using stochastic gradient descent for larger datasets.\n",
     "\n",
-    "**Note**: For larger datasets, this means integrating MiniRocket with stochastic gradient descent such that the transform is performed per minibatch, *not* simply substituting `RidgeClassifierCV` for, e.g., `LogisticRegression`."
+    "**Note**: For larger datasets, this means integrating MiniRocket with stochastic gradient descent such that the transform is performed per minibatch, *not* simply substituting `RidgeClassifierCV` for, e.g., `LogisticRegression`.\n",
+    "**Note**: While the input features of MiniRocket are unscaled, we recommend scaling the features when passing them to another classifier such as `RidgeClassifierCV`."
```

Comment:
Is this something recommended for all classifiers? I know the default RidgeClassifierCV normalises, but do they recommend it for others as well?

Reply:
True, was not meant to be an editorial recommendation. However, `RidgeClassifierCV(normalize=True)` has now a deprecated `normalize` arg, which is why we may use e.g. StandardScaler() in this tutorial for forward-compatability.

As in https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.RidgeClassifierCV.html
`Deprecated since version 1.0: normalize was deprecated in version 1.0 and will be removed in 1.2. `

Reply:
I think the note is fine now. We should probably use `StandardScaler(with_mean=False)` to keep it consistent with the classifier, however.